docs/extensions/installing-extensions.md

```diff
@@ -0,0 +1,139 @@
+---
+contributors: Esther Agbaje
+description: Discover how to install extensions into your Directus project.
+---
+
+# Installing Extensions
+
+There are two possible ways to install extensions to your Directus instance:
+
+1. Installing through NPM
+2. Installing through the Extensions Folder
+
+## Installing through NPM
+
+Package extensions are essentially npm packages that can be installed from the npm registry. Before you begin, ensure
+you have a [self-hosted instance of Directus](/self-hosted/quickstart) via Docker installed on your system.
+
+**1. Modify docker-compose.yml**
+
+Open the `docker-compose.yml` file of your project and modify the build section:
+
+```yaml
+build:
+  context: ./
+```
+
+This allows Docker to build your project with the necessary dependencies.
+
+**2. Create a Dockerfile**
+
+At the root of your project, create a `Dockerfile` if one doesn't already exist and add the following:
+
+```Dockerfile
+FROM directus/directus:latest
+
+USER root
+RUN corepack enable \
+ && corepack prepare pnpm@8.3.1 --activate
+
+USER node
+RUN pnpm install directus-extension-package-name
+```
+
+::: tip Extension Name
+
+Remember to replace `directus-extension-package-name` with the name of the extension you want to install. For example,
+`directus-extension-myextension`.
+
+:::
+
+**3. Build the Docker Image**
+
+Build your Docker image:
+
+```bash
+docker compose up --build
+```
+
+**4. Start the Docker Container**
+
+Start your Docker container:
+
+```bash
+docker compose up
+```
```

Comment:
`docker compose up --build` already starts the container so these commands should either be combined or otherwise the first one should be replaced with `docker compose build`.